robot/Cumulus/tests/browser/gift_entry/reorder_template.robot

```diff
@@ -0,0 +1,67 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/NPSPSettingsPageObject.py
+...             robot/Cumulus/resources/GiftEntryPageObject.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             API Check And Enable Gift Entry
+...             Setup Test Data
+Suite Teardown  Capture Screenshot and Delete Records and Close Browser
+
+*** Variables ***
+
+
+*** Keywords ***
+
+Setup Test Data
+    ${NS} =              Get NPSP Namespace Prefix
+    Set suite variable   ${NS}
+
+*** Test Cases ***
+
+Reorder and Modify GE Template Fields
+  [Documentation]                       This tests reordering, adding, and deleting sections
+  ...                                   from a newly created GE template.
+
+  [tags]                                unstable                    feature:GE          W-039563
+  ${template} =                         Generate Random String
+  Go to Page                            Landing                     npsp__GE_Gift_Entry
```

Comment:
u dont need to specify namespace, it should just be GE_Gift_Entry

Reply:
This has been resolved.